src/metrics/router_metrics.erl

```diff
@@ -510,6 +511,21 @@ record_queues() ->
     ),
     ok.
 
+-spec record_hotspot_reputations() -> ok.
+record_hotspot_reputations() ->
+    case router_hotspot_reputation:enabled() of
+        false ->
+            ok;
+        true ->
+            lists:foreach(
+                fun({Hotspot, Counter}) ->
+                    Name = blockchain_utils:addr2name(Hotspot),
+                    _ = prometheus_gauge:set(?METRICS_HOTSPOT_REPUTATION, [Name], Counter)
```

Comment:
We're currently running 15k unique on prod.
Prometheus doesn't like over 20k values for labels.

Reply:
This should not be that many hotspots, maybe we can set a limit?